epochStart/bootstrap/process.go

```diff
@@ -950,6 +992,15 @@ func (e *epochStartBootstrap) requestAndProcessForShard(peerMiniBlocks []*block.
 		return epochStart.ErrWrongTypeAssertion
 	}
 
+	ctx, cancel = context.WithTimeout(context.Background(), DefaultTimeToWaitForRequestedData)
```

Comment:
What is DefaultTimeToWaitForRequestedData ? Here we are requesting a few blocks, which might take a little bit more time. But if it is in minutes, then it is ok.

Reply:
It's one minute, I think it should be ok.

```go
const DefaultTimeToWaitForRequestedData = time.Minute
```